src/MOIWrapper.jl

```diff
@@ -368,3 +368,13 @@ end
 function LQOI.delete_variables!(instance::Optimizer, start_col::Int, end_col::Int)
     delete_columns(instance.inner, [Cint(i-1) for i in start_col:end_col])
 end
+
+function LQOI.add_mip_starts!(instance::Optimizer, cols::Vector{Int},
+                              x::Vector{Float64})
+
```

Comment:
Maybe I should check the size of cols before, since CLP doesn't support apparently partial initial solutions?

Reply:
Yes, it would work if LQOI guarantees that there is no duplicates in the cols. Otherwise, you can use a BitSet